Makefile

```diff
@@ -0,0 +1,27 @@
+all:
+	@echo 'MakeFile for DataJoint packaging                                              '
```

Comment:
would be cool if you could edit this line and replace `DataJoint` with the name of the package.